### PR TITLE
Add support for multiple channels

### PR DESCRIPTION
**Pull request by aermakov-zalando**
`stups_auto_configuration`: fetch and configure `LatestTaupageStagingImage` and `LatestTaupageDevImage` as well.

### CODE REVIEW

**Comment by mikkeloscar** [comment]
Should this maybe be documented in the README?

**Comment by aermakov-zalando** [comment]
@mikkeloscar I'll create a separate PR to [STUPS documentation](https://github.com/zalando-stups/documentation) later. The existing README doesn't have a lot of Taupage documentation, so I don't think it makes sense to document this change here.

**Comment by mikkeloscar** [comment]
:+1:

File: senza/components/taupage_auto_scaling_group.py [hunk 1]
```diff
@@ -18,6 +18,11 @@
 from typing import Optional
 
 _AWS_FN_RE = re.compile(r"('[{]{2} (.*?) [}]{2}')", re.DOTALL)
+_CHANNEL_MAPPING = {
+    "stable": "LatestTaupageImage",
```
Comment (jmcs) [inline senza/components/taupage_auto_scaling_group.py]: Are the names `LatestTaupageImage`, `LatestTaupageStagingImage` and `LatestTaupageDevImage` used outside senza code? Why not just use `stable`, `staging` and `dev`?

Comment (aermakov-zalando) [reply senza/components/taupage_auto_scaling_group.py]: `LatestTaupageImage` is used written into the generated CF template, so it's possible that someone else depends on it. I don't think it's a good idea to break backwards compatibility over a trivial rename.

**Comment by szuecs** [comment]
@jmcs does it requires another change?
For me this looks good

**Comment by aermakov-zalando** [comment]
I've removed the `TaupageChannel` property from the `TaupageAutoScalingGroup` component. `Image: LatestTaupageStagingImage` is a bit more verbose, but this is an already documented way of selecting an AMI.

**Comment by aermakov-zalando** [comment]
👍

**Comment by jmcs** [comment]
👍

**Pull request merged**